pkgs/servers/monitoring/fusion-inventory/default.nix

```diff
@@ -4,29 +4,15 @@
 
 perlPackages.buildPerlPackage rec {
   pname = "FusionInventory-Agent";
-  version = "2.3.21";
+  version = "2.5.2";
 
   src = fetchFromGitHub {
     owner = "fusioninventory";
     repo = "fusioninventory-agent";
     rev = version;
-    sha256 = "034clffcn0agx85macjgml4lyhvvck7idn94pqd2c77pk6crvw2y";
+    sha256 = "12grxf56kha71p9rd4p6kfp247c91skv5l82ckjyis7lz3m2dsqc";
   };
 
-  patches = [
-    ./remove_software_test.patch
-    # support for os-release file
-    (fetchurl {
```

Comment:
Remove fetchurl from inputs.

Reply:
But the _are_ removed. I removed the complete `patches = [ ... ];` attribute. You were commenting on the removed part of the patch. ;)

Reply:
I mean the the fetchurl in line 1.

Reply:
Done.